notebooks/query-language-object-based-dialect/01.01.quantifier_capabilities.ipynb

```diff
@@ -0,0 +1,842 @@
+    "\n",
+    "A valid hatchet query requires a **quantifier**. The accepted values for query node quantifiers in the **Object-based Dialect** are:\n",
+    "\n",
+    "1. `\".\"`: Match a single node\n",
+    "2. `\"*\"`: Match 0 or more nodes\n",
+    "3. `\"+\"`: Match 1 or more nodes\n",
+    "4. `Integer`: Match an exact number of nodes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb2db956",
+   "metadata": {},
+   "source": [
+    "### Loading profile data as Hatchet GraphFrame\n",
+    "\n",
+    "Hatchet queries are only defined on Hatchet GraphFrames. \n",
+    "Obtaining a hatchet GraphFrame is straight forward:\n",
+    "\n",
+    "1. Import hatchet\n",
+    "2. Use the appropriete reader for the profile/trace at hand\n",
+    "\n",
+    "This notebook loads a [Caliper](https://github.com/LLNL/Caliper) profile in JSON format and Caliper is a performance profiling library.\n",
+    "\n",
+    "The dataset contains [LULESH (Livermore Unstructured Lagrangian Explicit Shock Hydrodynamics)](https://asc.llnl.gov/codes/proxy-apps/lulesh) performance report data generated by Caliper. LULESH is a highly simplified application designed to solve the Sedov Blast problem, which is a standard hydrodynamics test problem. It performs a hydronomics stencil calculation using both MPI and OpenMP to achieve parallelism. \n",
+    "\n",
+    "This is an interesting profile because it covers a relatively large number of nodes and also spends considerable time in MPI communication routines."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "04ab3f46",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hatchet as ht\n",
+    "gf = ht.GraphFrame.from_caliper(\"../../data/lulesh-16nodes/lulesh-annotation-profile-512cores.json\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f37a13b-453e-4107-ac1c-b6025e390541",
+   "metadata": {},
+   "source": [
+    "### Displaying a Hatchet GraphFrame\n",
+    "A compact overview of a hatchet GraphFrame can be obtained using the `gf.tree()` function. We use this throughout the notebook to display the differences between an original GraphFrame and the resulting GraphFrame after applying a query."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4d29e959",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(gf.tree())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "152c81ff-4c15-4b5a-a7ba-6cb2c44d7479",
+   "metadata": {},
+   "source": [
+    "### Displaying a DataFrame\n",
+    "An additional detail perspective can be obtained by viewing the underlying data using a **DataFrame**. Hatchet uses pandas **DataFrames** to store the data on each node of the hierarchy and keeps the graph relationships between the nodes in a different data structure that is kept consistent with the **DataFrame**. The **DataFrame** holds all the numerical and categorical data associated with each node. "
```

Comment:
I am unsure if a detailed explanation of [Hatchet.DataFrame] is required given the context of the notebook. If needed, I would probably make the explanation on this cell terser.